src/ASBNApp.Frontend/Data/Professions.cs

```diff
@@ -0,0 +1,18 @@
+namespace ASBNApp.Frontend.Data;
+
+public static class Professions {
+
+    // Include all available job titles here
+    public static List<string> Apprenticeships = new()
+    {
+        "IT-System-Kaufmann/-frau",
+        "Kaufmann/-frau für IT-System-Management",
+        "IT-System-Elektroniker/-in",
+        "Kaufmann/-frau für Büromanagement",
+        "Kaufmann/-frau im Einzelhandel",
+        "Fachinformatiker für Anwendungsentwicklung",
+        "Fachinformatiker für Systemintegration",
+        "Kaufmann/-frau für Dialogmarketing",
+        "Industriekaufmann/-frau"
```

Comment:
What would be a funny feature request, is to fetch all the education paths from IHK, so this list gets never old

Reply:
Oh hell yeah - but before I start messing with the IHK guys, I'll add the dice feature, that let's you "roll a dice" and then adds a random generic thing like "Bugfixing on the API". Could be helpful if you forgot to track what you did for some time for example... :D